### ISSUE TITLE
Format and mount the data-volume

### ISSUE THREAD

**Issue by afbjorklund** [afbjorklund/tinycloud#3]
The default script needs `util-linux` package, for `lsblk` and `sfdisk`

```
252K	/tmp/tce/optional/ncursesw.tcz
144K	/tmp/tce/optional/readline.tcz
24K	/tmp/tce/optional/udev-lib.tcz
1.4M	/tmp/tce/optional/util-linux.tcz
```

https://github.com/lima-vm/lima/blob/master/pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh

The disk is there already (`/dev/vda`), just not formatted and mounted.

**Comment by afbjorklund** [comment]
The missing `/dev/disk/by-label` symlinks, were caused by missing `udev-extra.tcz`

**Comment by afbjorklund** [comment]
Fixed, `v0.14`
